ssm/ssm2.py

```diff
@@ -36,6 +36,8 @@
 import time
 import logging
 
+from argo_ams_library import ArgoMessagingService, AmsMessage, AmsException
```

Comment:
Prob only want either AMS or STOMP needed to be installed. Use SSL type check.

Reply:
For simplicity of the code, I think we should postpone making it an either/or choice and just require both for the moment.